### PR TITLE
remove kivy specification file along with component changes of versio…

### PR DESCRIPTION
**Pull request by surbhicis**
remove buildozer.spec
remove android_instruction.rst
changes made on kivy components for disable kivy from this version

### CODE REVIEW

File: src/bitmessagemain.py [hunk 1]
```diff
@@ -353,8 +353,6 @@ def start(self):
                 bitmessagecurses.runwrapper()
             elif state.kivy:
```
Comment (g1itch) [inline src/bitmessagemain.py:354]: Also remove this expression

**Comment by PeterSurda** [comment]
Should probably be marked as Draft until the issues are fixed.

I requested this work to be done, it's so that kivy support can be re-added with proper code quality and tests.

**Comment by PeterSurda** [comment]
This PR should be changed to Draft until cleaned up. I requested the work to be done, so that kivy can be re-added later, with proper code quality and tests.

**Review by g1itch — APPROVED** [review]

**Review by PeterSurda — APPROVED** [review]
Looks ok to me

**Comment by g1itch** [comment]
Please rebase

**Pull request merged**